src/App.js

```diff
@@ -9,16 +14,28 @@ import './App.css';
 import List from './pages/List/List';
 import AddItem from './pages/AddItem/AddItem';
 import NotFound from './pages/NotFound/NotFound';
+import Home from './pages/Home/Home';
 
 // Components
 import NavMenu from './components/NavMenu/NavMenu';
 
 function App() {
+  const [haveToken, setHaveToken] = useState(false);
+
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (token) {
+      setHaveToken(true);
+    }
+  }, []);
   return (
     <Router>
       <div className="App container">
         <Switch>
           <Route exact path="/">
+            {haveToken ? <Redirect to="/list" /> : <Home />}
+          </Route>
+          <Route path="/list">
             <List />
           </Route>
           <Route path="/add">
```

Comment:
The way the AC below is written it's not clear what happens when a user does not have a token saved in local storage. 
> For users who do have a token already saved in localStorage, they should be shown/redirected to the "list" view

The way to interpret the AC is:
- if the user navigates to https://tcl-26-shopping-list.web.app and they do have a token, then they are redirected to List view
- if the user does not have a token, they should not be able to view the list or add an item until they create a token
  - that means if a user navigates directly to https://tcl-26-shopping-list.web.app/list or https://tcl-26-shopping-list.web.app/add, they should be redirected to home to create a new token. 

The way the routes are set up, the above AC is not met and creates a bug.